{ 'test_3': { 'left_arm': [],
              'left_leg': [ { 'duration': 3.002,
                              'name': u'high_step_6_left',
                              'positions': [ 0.01,
                                             -0.1576,
                                             -1.46,
                                             2.0629,
                                             0.8216,
                                             0.02],
                              'starttime': 0.0}],
              'neck': [],
              'right_arm': [],
              'right_leg': [ { 'duration': 3.002,
                               'name': u'high_step_6_right',
                               'positions': [ 0.02,
                                              -0.0337,
                                              -0.1525,
                                              0.0301,
                                              -0.3529,
                                              0.1286],
                               'starttime': 0.0}],
              'torso': []}}
